specificatie/filter.yaml

```diff
@@ -30,31 +30,31 @@ components:
       example: "V"
     GeslachtsnaamFilter:
       description: |
-        De (geslachts)naam waarvan de eventueel aanwezige voorvoegsels zijn afgesplitst. **Gebruik van de wildcard is toegestaan. Zie [feature-beschrijving](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/wildcard.feature)** **Zoeken met tekstvelden is [case-Insensitive](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.3.0/features/case_insensitive.feature).**
+        De (geslachts)naam waarvan de eventueel aanwezige voorvoegsels zijn afgesplitst. **Gebruik van de wildcard is toegestaan bij invoer van ten minste 3 letters.** **Zoeken met tekstvelden is case-Insensitive.**
```

Comment:
`case-Insensitive` hernoemen naar `case-insensitive`?

Reply:
geen idee hoe die hoofdletter I daar gekomen is, maar heb het gecorrigeerd.